telemetry-testing/src/test/java/io/opentelemetry/example/telemetry/ApplicationTest.java

```diff
@@ -55,7 +56,9 @@ class ApplicationTest {
 
   @BeforeAll
   public static void setUp() {
-    collectorServer = startClientAndServer(EXPORTER_ENDPOINT_PORT);
+    Configuration config = Configuration.configuration();
+    config.localBoundIP("127.0.0.1");
+    collectorServer = startClientAndServer(config, EXPORTER_ENDPOINT_PORT);
```

Comment:
could this change be related to the build failures?

Reply:
Maybe? I had issues locally, but I'll remove it and see.

Reply:
I'll give it a spin locally on Windows